src/Offer/Offer.php

```diff
@@ -720,6 +725,20 @@ protected function importWorkflowStatus(CultureFeed_Cdb_Item_Base $cdbItem): voi
         $this->workflowStatus = $workflowStatus;
     }
 
+    public function updateAvailableFrom(DateTimeInterface $availableFrom): void
+    {
+        if ($availableFrom < new DateTimeImmutable()) {
+            $availableFrom = new DateTimeImmutable();
```

Comment:
As far as I can tell, there is no test to verify this behaviour.

Reply:
Indeed no test for this because the scenario's work with an `assertEquals` and we don't have a way to mock the `now`. I will try to fix this...

Reply:
I found a way to test this by storing the traceable event store and comparing the recorded event manually and making sure the time difference is small.